core/src/main/java/com/radixdlt/RadixNodeModule.java

```diff
@@ -425,6 +428,11 @@ protected void configure() {
         properties.get("api.engine_state.port", DEFAULT_ENGINE_STATE_API_PORT);
     install(new EngineStateApiServerModule(engineStateApiBindAddress, engineStateApiPort));
 
+    final var meshApiBindAddress =
```

Comment:
Could we also update `default.config.envsubst` with this new configuration? :)

Reply:
Done